openshift/openshift-ansible-contrib
# Ignore *.cfg and *.crt in the openstack inventory

#### What does this PR do?
This allows our users to keep the ansible.cfg file in the inventory as
well as putting e.g. LDAP certificates in.

Fixes #481

#### How should this be manually tested?
Put a file ending with *.crt (e.g. the LDAP CA certificate) in the inventory directory alongside the `group_vars` dir.

Run the openstack provision.yaml playbook and verify that it succeeds.

#### Is there a relevant Issue open for this?
#481 

#### Who would you like to review this?
cc: @Tlacenka @tzumainn @rlopez133  @bogdando PTAL
